sys/fido2/ctap/ctap_mem.c

```diff
@@ -154,3 +166,18 @@ int fido2_ctap_mem_get_offset_of_rk_into_flashpage(uint16_t rk_idx)
 
     return -1;
 }
+
+unsigned fido2_ctap_mem_get_state_flashpage_number(void)
+{
+#if defined(CONFIG_FIDO2_CTAP_NUM_FLASHPAGES) && \
```

Comment:
Should this be `CONFIG_FIDO2_CTAP_FLASH_NUM_FLASHPAGES` (generated from [here](https://github.com/RIOT-OS/RIOT/pull/17096/files#diff-2c0cb8c9ff04ebbb2b2d14a8c59f79e75e94fad8e56e96dcc1555a32147cd614R87))? Btw, prefer the name `CONFIG_FIDO2_CTAP_NUM_FLASHPAGES`.